### PR TITLE
[SD-1327] modified the code for import profile for AD.

### PR DESCRIPTION
**Pull request by mdhaman**

### CODE REVIEW

File: apps/auth/ldap/ldap.py [hunk 1]
```diff
@@ -122,6 +125,17 @@ def authenticate_and_fetch_profile(self, username, password, username_for_profil
 
 class ADAuthService(AuthService):
 
+    def on_create(self, docs):
+
+        user_service = get_resource_service('users')
+        for index, doc in enumerate(docs):
```
Comment (amagdas) [inline apps/auth/ldap/ldap.py]: `index` is not used so no need for enumerate.

File: apps/auth/service.py [hunk 2]
```diff
@@ -15,6 +15,9 @@ def on_create(self, docs):
             if not user.get('is_active', False):
                 raise UserInactiveError()
 
-            doc['user'] = user['_id']
-            doc['token'] = utils.get_random_string(40)
-            del doc['password']
+            self.set_auth_default(doc, user['_id'])
+
+    def set_auth_default(self, doc, id):
+        doc['user'] = id
```
Comment (amagdas) [inline apps/auth/service.py]: We should avoid using built-in functions as variable names, at least to not cause confusion.
`id` is a built-in function: https://docs.python.org/2/library/functions.html#id

**Comment by amagdas** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**